README.md

```diff
@@ -25,7 +25,7 @@ Welcome to the home of the TANF Data Portal (TDP), a new software development pr
 Both the frontend (`http://localhost:3000`) and the backend (`http://localhost:8080`) applications run within Docker.  Instructions for running these containers are below:
 
 ```
-$ cd tdrs-frontend && docker-compose up -d
+$ cd tdrs-frontend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d
```

Comment:
@jorgegonzalez should the `-d` be removed in these instructions?

Reply:
I don't think it should be removed as all our other documented `docker-compose` commands use `-d` to run the containers in the background, but it *is* an optional flag.